cobc/pplex.l

```diff
@@ -1867,6 +1882,19 @@ int cobc_deciph_source_format (const char *sfname) {
 	return 1;
 }
 
+int cobc_deciph_header_filename (const char *filename) {
+	size_t n = strlen(filename);
+	if (filename[0] == '<' && filename[n-1] == '>' &&
+      filename[n-3] == '.' && filename[n-2] == 'h') {
+			return 0;
+	} else if (filename[n-2] == '.' && filename[n-1] == 'h') {
+		return 0;
+	}
+	else {
+		return 1;
+	}
+}
```

Comment:
That is validation, not deciphering; also: is there a reason to limit to .h files? I'd just pass as-is - we do the same for `--include`, no?`
If the validation should stay in then move to the single file that actually uses this, rename it and check n before breaking with `>>IMP INCLUDE "no"` (access out of bounds)